tmdb.py

```
@@ -0,0 +1,78 @@
+        list_id: TMDb list ID (numeric or alphanumeric).
+        api_key: TMDb API Key (v3).
+
+    Returns:
+        An ordered list of TMDb IDs (as strings).
+
+    Raises:
+        ValueError: If *api_key* or *list_id* is empty.
+        RuntimeError: If an HTTP error occurs while fetching a page.
+    """
+    if not api_key:
+        raise ValueError("A TMDb API Key is required to fetch TMDb lists.")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)